mos-platform/common/lib/__cxxabi_config.h

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ____CXXABI_CONFIG_H
+#define ____CXXABI_CONFIG_H
+
+#if defined(__arm__) && !defined(__USING_SJLJ_EXCEPTIONS__) &&                 \
```

Comment:
Can we clean this up a bit, take out the stuff that "obviously" doesn't apply? That would make it quite a bit easier to review, since I'm going to have to learn how this algorithm works.

Reply:
OK; I removed the sections that didn't look like they'd ever be compiled in.